src/ZenithAngleCalc.jl

```diff
@@ -114,36 +116,43 @@ end
     daily_zenith_angle(date::DateTime,
                        latitude::FT,
                        param_set::APS;
-                       eot_correction::Bool=true) where {FT <: Real}
+                       eot_correction::Bool=true,
+                       milankovitch::Bool=true) where {FT <: Real}
+
 Returns the daily averaged zenith angle and earth-sun distance
```

Comment:
```suggestion
Returns the effective zenith angle corresponding to diurnally averaged insolation and earth-sun distance
```
(because what's averaged is the cosine of the zenith angle)